server.py

```diff
@@ -67,6 +68,8 @@
 def create_app() -> Flask:
     app = Flask(__name__)
     app.url_map.strict_slashes = False
+    app.debug = DEBUG
+    os.environ["FLASK_DEBUG"] = str(DEBUG)
```

Comment:
do you know what changes if debug is set to true?

Reply:
you mean the first or the second line?

Reply:
I mean the second line. This setting is used by `flask cli` I think and as `server.py` is not launched via `flask cli`, I'm not sure that setting could have any effect.

Reply:
(y) i rolled back these flags, it is indeed has no effect with the server script